tests/reporter/kubernetes.go

```diff
@@ -66,23 +64,25 @@ const (
 )
 
 type KubernetesReporter struct {
-	failureCount int
-	artifactsDir string
-	maxFails     int
-	namespace    string
+	failureCount    int
+	artifactsDir    string
+	maxFails        int
+	namespaces      []string
+	isRunningOnKind bool
 }
 
 type commands struct {
 	command        string
 	fileNameSuffix string
 }
 
-func NewKubernetesReporter(artifactsDir string, maxFailures int, namespace string) *KubernetesReporter {
+func NewKubernetesReporter(artifactsDir string, maxFailures int, namespaces []string, isRunningOnKind bool) *KubernetesReporter {
```

Comment:
pedantic nit: I would avoid the boolean arg, and add two constructor funcs instead, `NewKubernetesReporter` and `NewKubernetesReporterForKindClusters`.